.github/files/setup-wordpress-env.sh

```diff
@@ -128,6 +128,43 @@ for PLUGIN in projects/plugins/*/composer.json; do
 	echo "::endgroup::"
 done
 
+# Install WooCommerce plugin used for some Jetpack integration tests.
+#DEFAULT_PHP_VERSION=$(bash -c 'source .github/versions.sh; echo $PHP_VERSION')
+if [[ "$WITH_WOOCOMMERCE" == true ]] &&
+	jq --argjson changed "$CHANGED" -ne '$changed["plugins/jetpack"] // false' > /dev/null
+then
+	echo "::group::Installing plugin WooCommerce into WordPress"
+
+	WOO_REPO_URL="https://github.com/woocommerce/woocommerce"
+	WOO_GH_API_URL="https://api.github.com/repos/woocommerce/woocommerce/releases/latest"
+	WOO_LATEST_TAG=""
+	WOO_DL_URL=""
+
+	RESPONSE=$(curl -s "$WOO_GH_API_URL")
+	WOO_LATEST_TAG=$(jq -r ".tag_name" <<< "$RESPONSE")
+	WOO_DL_URL=$(jq -r ".assets[0].browser_download_url" <<< "$RESPONSE")
```

Comment:
It doesn't make any real difference, but in general it's more useful to use single quotes here
```suggestion
	WOO_LATEST_TAG=$(jq -r '.tag_name' <<< "$RESPONSE")
	WOO_DL_URL=$(jq -r '.assets[0].browser_download_url' <<< "$RESPONSE")
```